### PR TITLE
Create recommendations

### PR DESCRIPTION
**Pull request by mocca102**

### CODE REVIEW

File: src/components/Recommendations/index.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import { useEffect, useState } from 'react';
+
+import { fetchRecommendations } from '../../utils';
+import { loadStatuses } from '../../utils/constants';
+// eslint-disable-next-line import/no-cycle
```
Comment (esezen) [inline src/components/Recommendations/index.jsx:7]: Do we still need this?

File: src/components/Recommendations/index.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import { useEffect, useState } from 'react';
+
```
Comment (esezen) [inline src/components/Recommendations/index.jsx:4]: Extra linebreak

File: src/components/Recommendations/index.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
```
Comment (esezen) [inline src/components/Recommendations/index.jsx:2]: Extra linebreak

**Comment by esezen** [comment]
Can we add a max width to the modal and the elements inside? I think it's getting really big on large screens. Can we also align the top part and the recommendation container and the contents of the recommendation container?
Screen Shot 2022-11-01 at 10 25 08 AM

On small screens, the recommendations stay under the fold but when you scroll, the body scrolls instead of the modal.
Screen Shot 2022-11-01 at 10 25 30 AM

**Comment by esezen** [comment]
Thanks for the changes! Desktop is looking good. Is there anything we can do about mobile?
Screen Shot 2022-11-08 at 10 14 54 AM
Also can we get another review from someone else from the team.

**Review by esezen — APPROVED** [review]
LGTM

**Review by sblaurock — APPROVED** [review]
Looks good to me! I made some minor visual tweaks which should help a bit. If those look OK to you, feel free to ship. Thanks!

Note that I disabled the modal functionality via media queries for widths below 500px. Not the most elegant solution, but avoids broken states.

I also disabled the carousel scrolling as it's not scrolling the correct width for me and felt broken. If we can fix that, we can restore the functionality.

Screenshot 2022-11-13 at 10 07 46 PM

**Pull request merged**